### PR TITLE
Fix polish translations

### PR DESCRIPTION
**Pull request by miedzikd**

### CODE REVIEW

**Comment by miedzikd** [comment]
Hi @MarvinZeising 
I edited some translations, after review with some brothers from my congregation...

Also I created translation for datepicker... (/client/lib/datepicker/bootstrap-datepicker.pl.js)
This can fix problem with screen below?
polish translations

File: client/views/pages/firstLogin/firstLogin.tpl.jade [hunk 1]
```diff
@@ -31,7 +31,7 @@
 												input#agreeTerms(type='checkbox')
 												label(for='agreeTerms') {{{_ 'firstLogin.agreeTerms'}}}
 										button.btn.btn-primary.block.full-width(type='submit') {{_ "firstLogin.button"}}
-										button#haveAccount.btn.btn-white.block.full-width.m-t I already have an account
+										button#haveAccount.btn.btn-white.block.full-width.m-t {{_ "firstLogin.buttonCreateAccount"}}
```
Comment (MarvinZeising) [inline client/views/pages/firstLogin/firstLogin.tpl.jade]: Shouldn't this be buttonHaveAccount?

Comment (miedzikd) [reply client/views/pages/firstLogin/firstLogin.tpl.jade]: Sure! thanks for tip!

**Comment by miedzikd** [comment]
Hmm... I very like your translation system!... for me is very good!

**Comment by miedzikd** [comment]
At 5.11.2017 in our congregation, we have special training for people who are public preaching. 
What do you think... You will deploy this changes before Sunday?

**Comment by MarvinZeising** [comment]
Good to know! I think so, yes.

**Comment by miedzikd** [comment]
Thanks @MarvinZeising ... :) please give me sign when you deploy this commit on production server :)

**Comment by MarvinZeising** [comment]
Has just been deployed :-)
Datepickers don't seem to work correctly, though.
I'll have a look at that later.

**Comment by miedzikd** [comment]
Thanks a lot my dear brother!

05.11.2017 9:32 AM "Marvin Zeising" <notifications@github.com> napisał(a):

Has just been deployed :-)
Datepickers don't seem to work correctly, though.
I'll have a look at that later.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/JWDeveloper/JWManagement/pull/170#issuecomment-341957021>,
or mute the thread

**Comment by PPdev** [comment]
Hi,

[this line](https://github.com/JWDeveloper/JWManagement/blob/3d72dca83b042832263e783c211f4aad2c69031c/client/lib/datepicker/bootstrap-datepicker.pl.js#L4) should be `    a.fn.datepicker.dates.pl = {`, I guess.
Could be the reason for my Polish datepicker in the German environment. ;-)

Kind regards
Pascal

**Comment by MarvinZeising** [comment]
Thank you very much, Pascal
Fix has been deployed

**Pull request merged**